### ISSUE TITLE
'Scroll Element Into View' is broken on Firefox

### ISSUE THREAD

**Issue by Brownies** [robotframework/SeleniumLibrary#1780]
## Steps to reproduce the issue
* Open a page in Firefox such that some element is not inside the viewport
* Try to scroll to the element with 'Scroll Element Into View'

## Expected behavior and actual behavior

When I follow those steps, I see...
[Gist with trace level log output, stack trace and sample .robot and .html files](https://gist.github.com/Brownies/72f9d5b42a14174bcfd25bb9080ec6ef)

I was expecting the element to be scrolled into view. This does work on Chromium, both headless and normal.

## Environment

Browser: Firefox 103.0
Browser driver: geckodriver 0.31.0
Operating System: Linux, Ubuntu 20.04
Libraries
- Robot Framework: 5.0
- Selenium: 4.1.3 / 4.4.3
- SeleniumLibrary: 6.0.0
- Interpreter: CPython 3.8.10

**Comment by Brownies** [comment]
A likely culprit seems to be the [keyword itself](https://github.com/robotframework/SeleniumLibrary/blob/c6e28bdda61abefefa875f4ef8661d7e2a6f81ca/src/SeleniumLibrary/keywords/element.py#L739) which uses `ActionChains.move_to_element` instead of `ActionChains.scroll_to_element`.

**Comment by everlighting** [comment]
Is some one following up? When this issue will be released?  Once this issue is fixed we can remove the JS in our script

**Comment by Brownies** [comment]
I'll see if I can fix this and get the acceptance tests running on Firefox too.

**Comment by Brownies** [comment]
Bad news, switching to `ActionChains.scroll_to_element` does not fix the problem. It's also present with RF 6 and selenium 4.7.2 with Firefox 108.
Maybe this is a Selenium or Geckodriver problem?

**Comment by Brownies** [comment]
Yeah, this is an ancient Geckodriver issue and they will not fix it until W3C updates the WebDriver spec: https://github.com/mozilla/geckodriver/issues/776#issuecomment-355086173
Might as well close this issue because the problem is not in SeleniumLibrary and judging by the rate of recent commits the spec update is going to take an eternity.
